frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBindings.tsx

```diff
@@ -88,15 +96,10 @@ export const EditDataModelBindings = ({
                 propertyPath={`definitions/component/properties/dataModelBindings/properties/${
                   key || 'simpleBinding'
                 }`}
-                label={
-                  label
-                    ? `${t('ux_editor.modal_properties_data_model_helper')} ${t(
-                        'general.for',
-                      )} ${label}`
-                    : t('ux_editor.modal_properties_data_model_helper')
-                }
```

Comment:
I think we need to keep this as-is, otherwise there is no way of differentiating when there are more than 1 data model for a component 🤔

Reply:
Ah okey, I just discovered that it is being rendered in english so I figured it was better to remove it. 

![Skjermbilde 2024-02-21 kl  08 38 42](https://github.com/Altinn/altinn-studio/assets/71079896/fb90812e-e030-4407-bdf3-e90afee4689f)

![Skjermbilde 2024-02-21 kl  08 41 34](https://github.com/Altinn/altinn-studio/assets/71079896/fe6a49d3-bbf4-4e34-b50a-bec1a4ca979b)